Assets/Dev/Arthur/ArthurAssets/Highlight Mechanic/Highlight Quad.prefab

```diff
@@ -0,0 +1,96 @@
+%YAML 1.1
```

Comment:
Can you add some documentation on how to use this mechanic? Putting it in the README would be great